labels_new.md

```diff
@@ -0,0 +1,221 @@
+# Labels Proposal
+
+## Goals
+
+* synchronise labels between repos, especially priority labels
+* make colours more useful by making them more uniform
+
+## Current issues
+
+* Priotiry labels settings needs to be synchronised among repos.
+
+* TiDB uses a `category/label` scheme, but with some exceptions
+
+- closed, component, priority, status, type
+
+## Proposal
+
+### Naming Aternatives
+
+* Use `category/label` naming schemes
+* More categories (e.g., a bot category)
+* Allow labels without categories
+* Add emoji to highlight some labels or sub-categories
+* Separate labels for issues vs PRs
+
+### The labels
+`*` indicates I think we should remove an issue, there might be a suggested replacement in `<...>`. `?` means I don't know what the label means.
+
+### Priotity lables
+
+* Priority labels synchronising proposal:
+
+```
+priority/critical-urgent        #cc2400        Highest priority. Must be actively worked on as someone's top priority right now. Release blocker.
+priority/important-longterm     #fe5733        Important over the long term, but may not be staffed and/or may need multiple releases to complete.
+priority/important-soon         #fe5733        Must be staffed and worked on either currently, or very soon, ideally in time for the next release.
+priority/backlog                #feca33        Higher priority than priority/awaiting-more-evidence.
+priority/awaiting-more-evidence #ffe599        Lowest priority. Possibly useful, but not yet enough support to actually get it done.
+needs-priority                  #cdcdcd        Indicates a PR lacks a `priority/foo` label and requires one.
+```
+
+* Repos current settings which need to optimize
+
+- TiDB
+
+```
+priority/individual-blocker
+Priority/P0
```

Comment:
We're now using "severities" like `severity/critical` just like TiKV, I suggest keeping them.

Reply:
Is it just for bug-fixes or all types?

Reply:
As we spoken, "severities" label is kept using in describing bug severities; here we are focusing in refine priority labels that could be mostly used in features.